modules/libcom/src/iocsh/iocsh.cpp

```diff
@@ -1367,6 +1368,35 @@ iocshEnvClear(const char *name)
     }
 }
 
+void epicsStdCall
+iocshSet(const char *name, const char * value)
```

Comment:
Please check for `NULL`, and if `name` is an empty string. Also, consider the return value of `macPutValue` and handle it using `iocshSetError()`.

I think `macPutValue` should check for `NULL` and empty name itself, but I checked and it doesn't. Well, somebody has to :)

Reply:
Speaking of `iocshSetError()`, I see now that this change removes a bunch of error handling calls that were added just yesterday. I believe you were making changes against an older version of the code. Please rebase so that only the desired changes are made part of this pull request.

Reply:
#507 may conflict with #508, #497, and perhaps #459.  @JJL772 fyi.

Reply:
... and yes, also 6ca716a77f24247e96952d1778bb1673600bb420.